src/training/main.py

```diff
@@ -133,6 +133,11 @@ def main(args):
     )
     random_seed(args.seed, args.rank)
 
+    if hasattr(model, "text") and hasattr(model.text, "tokenizer"):
```

Comment:
not sure if it's necessary to check both. Can we expect that CustomTextCLIP will have the .tokenizer attribute?